rasa/cli/run.py

```diff
@@ -94,7 +94,7 @@ def run(args: argparse.Namespace):
     # make sure either a model server, a remote storage, or a local model is
     # configured
 
-    from rasa.model import get_model
+    from rasa.model import verify_model_path, get_model
```

Comment:
small note: Our Python [guide line](https://www.notion.so/rasa/Python-Code-Conventions-26835ffd72484dd699f28fa693916d2e#70d0247fdd1f48d2aec341fd91fe1dac) states that we rather import the modules instead of the functions directly. This makes it a little bit easier to see where a function is coming from when reading the code.